runtime/src/bank.rs

```diff
@@ -10227,19 +10185,19 @@ pub(crate) mod tests {
             if bank.slot == 32 {
                 assert_eq!(
                     bank.hash().to_string(),
-                    "4syPxVrVFUpksTre5BB5w7qd3BxSU4WzUT6R2fjFgMJ2"
+                    "7AkMgAb2v4tuoiSf3NnVgaBxSvp7XidbrSwsPEn4ENTp"
```

Comment:
For later reference, this is safe because I'm aware the specific behaviors of rent collection is changed by making some gated behavior now default.